### PR TITLE
Add sluggable candidate

### PR DESCRIPTION
**Pull request by modpluz**
- Add sluggable candidates
- Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Initially copied from Illinois Sunshine.
 
 ## Setup
 
-**Install OS level dependencies:** 
+**Install OS level dependencies:**
 
 * Python 3.4
 * PostgreSQL 9.4 +
```
Comment (lottspot) [inline README.md:10]: I mean, if you're gonna fix the README anyways... maybe you can change this PostgreSQL requirement to MySQL for us too 😉

File: leverageapi/api.py [hunk 2]
```diff
@@ -107,6 +107,7 @@ def races():
     response_str = json.dumps(resp, sort_keys=False, default=dthandler)
     response = make_response(response_str, 200)
     response.headers['Content-Type'] = 'application/json'
+    # response.headers['Access-Control-Allow-Origin'] = '*'
```
Comment (lottspot) [inline leverageapi/api.py:110]: why

File: leverageapi/api.py [hunk 3]
```diff
@@ -120,14 +121,21 @@ def candidates():
...
             .join(Candidate.candidacies)\
-            .filter(Candidate.id==candidate_id) 
+            .filter(Candidate.id==candidate_id)
+
+    elif 'slug' in request.args:
+        candidate_slug = request.args['slug']
+
+        candidates = db_session.query(Candidate)\
+            .join(Candidate.candidacies)\
+            .filter(Candidate.slug==candidate_slug)
```
Comment (lottspot) [inline leverageapi/api.py:138]: Seems like a sensible addition to me

File: README.md (hunk 1 above)
Comment (modpluz) [reply README.md:10]: lolz, I changed the first thing my eye caught.

File: leverageapi/api.py (hunk 2 above)
Comment (modpluz) [reply leverageapi/api.py:110]: Well, the api wouldn't work for me locally otherwise and this was the quickest fix I could muster.

Comment (lottspot) [reply leverageapi/api.py:110]: should be fixed by #66

**Comment by enactdev** [comment]
Looks good, but maybe use candidate_slug instead of just slug. Race can also have a slug,

**Pull request merged**